### PR TITLE
gh-100972: For command line examples in the documentation, use "python" rather than "python3"

### PR DESCRIPTION
**Pull request by furkanonder**
* Issue: gh-100972

### CODE REVIEW

**Review by hugovk — APPROVED** [review]
Thanks!

File: Doc/library/importlib.metadata.rst [hunk 1]
```diff
@@ -73,7 +73,7 @@ something into it:
 
 .. code-block:: shell-session
 
-    $ python3 -m venv example
```
Comment (felixxm) [inline Doc/library/importlib.metadata.rst]: I think we should leave `python3` here as it is an instruction how to create a virtual environment. I didn't change it on purpose.

File: Doc/library/__main__.rst [hunk 2]
```diff
@@ -259,7 +259,7 @@ one mentioned below are preferred.
 
    See :mod:`venv` for an example of a package with a minimal ``__main__.py``
    in the standard library. It doesn't contain a ``if __name__ == '__main__'``
-   block. You can invoke it with ``python3 -m venv [directory]``.
+   block. You can invoke it with ``python -m venv [directory]``.
```
Comment (felixxm) [inline Doc/library/__main__.rst]: I think we should leave `python3` here as it is an instruction how to create a virtual environment. I didn't change it on purpose.

File: Doc/tutorial/venv.rst [hunk 3]
```diff
@@ -44,7 +44,7 @@ whichever version you want.
 To create a virtual environment, decide upon a directory where you want to
 place it, and run the :mod:`venv` module as a script with the directory path::
 
-   python3 -m venv tutorial-env
+   python -m venv tutorial-env
```
Comment (felixxm) [inline Doc/tutorial/venv.rst]: I think we should leave `python3` here as it is an instruction how to create a virtual environment. I didn't change it on purpose.

File: Doc/whatsnew/3.9.rst [hunk 4]
```diff
@@ -222,7 +222,7 @@ Other Language Changes
   (Contributed by Ngalim Siregar in :issue:`37444`.)
 
 * Python now gets the absolute path of the script filename specified on
-  the command line (ex: ``python3 script.py``): the ``__file__`` attribute of
+  the command line (ex: ``python script.py``): the ``__file__`` attribute of
```
Comment (felixxm) [inline Doc/whatsnew/3.9.rst:225]: I don't see much value in changing old release notes.

**Comment by vsajip** [comment]
Doesn't this assume that the system Python installation will be Python 3.x such that "python" will start a 3.x interpreter? If that's the case, it shouldn't be needed to specify "python3" for "-m venv" usages, right? Also, is it definitely the case that "python" will invoke 3.x on all platforms, old and new? For example I have some (not very) old Ubuntu machines where "python3" is needed. Anyone following along the documentation with this change could run into problems.

**Comment by rhettinger** [comment]
On my Mac, `python` no longer refers to an executable. Only `python3` works.  So, at least for Mac Users, the status quo is better.  With the proposed substitutions, the examples won't work any more.

File: Doc/library/importlib.metadata.rst (hunk 1 above)
Comment (MonadChains) [reply Doc/library/importlib.metadata.rst]: I agree since older systems may still have Python 2.7 set a `python`.

**Comment by furkanonder** [comment]
> On my Mac, `python` no longer refers to an executable. Only `python3` works. So, at least for Mac Users, the status quo is better. With the proposed substitutions, the examples won't work any more.

If there are problems with the Mac, I think [this PR ](https://github.com/python/cpython/pull/98761) should be reverted.

**Pull request closed without merging**